packages/vscode/CONTRIBUTING.md

```diff
@@ -34,8 +34,8 @@ On push to the `main` branch, a new Insider extension is released, with an incre
 
 ## Development
```

Comment:
This now probably needs a not that pnpm has to be installed globally.

Reply:
I had to revert everything since vsce doesn't work with anything else than npm or yarn.
There is a solution to use pnpm and bundle using esbuild for example, while that work for the vscode extension I could not make it work with the language-server (the extension would ship without)